src/axolotl/utils/models.py

```diff
@@ -70,7 +71,7 @@ def load_tokenizer(
 def load_model(
     base_model, base_model_config, model_type, tokenizer, cfg, adapter="lora"
 ):
-    # type: (str, str, str, AutoTokenizer, DictDefault, Optional[str]) -> Tuple[PreTrainedModel, Optional[PeftConfig]]
+    # type: (str, str, str, PreTrainedTokenizerBase, DictDefault, Optional[str]) -> Tuple[PreTrainedModel, Optional[PeftConfig]]
```

Comment:
Ah, is this for the Fast tokenizer?

Reply:
no particular one, I just noticed the IDE wasn't properly suggesting the correct things.